scripts/comparator/run.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+set -ex
+export COMPARATOR=$GOPATH/src/github.com/m3db/m3/scripts/comparator
+source $COMPARATOR/docker-setup.sh
+
+export REVISION=$(git rev-parse HEAD)
+
+CI=${CI:-true}
+RUN_ONLY=${RUN_ONLY:-false}
+
+export QUERY_FILE=$COMPARATOR/queries
+export END=$(date +%s)
+export START=$(( $END - 10800 ))
```

Comment:
Might want to parameterize these too perhaps? `START=${START:-$(( $END - 10800 ))}` etc?

Reply:
Sure, sounds good... was planning to add some randomness to start/end/offset/step sizes eventually just to try and hit any overlooked cases eventually when I had spare time

Reply:
I see, yeah I was just wondering if users might also to override the default value?  i.e. do specific lookback/etc.

Reply:
Yeah this should be updated already